app/src/main/java/com/pajato/android/gamechat/exp/ChessPiece.java

```diff
@@ -7,7 +7,7 @@
 import java.util.ArrayList;
 
 /**
- * A simple P.O.J.O. class that keeps track of a chess pieces type and the team it is on.
+ * A simple P.O.J.O. class that keeps track of a chess_exp pieces type and the team it is on.
```

Comment:
One liner comments are accepted style and a good way to achieve conciseness without sacrificing readability. btw the standard AOSP line length is 100 characters in case you were not aware.